Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.py

```diff
@@ -0,0 +1,437 @@
+                    # Authentication data do not have breaches
+                    markdown.append("### Breaches")
+                    markdown.extend(breaches_section)
+                    markdown.append("*****")
+                markdown.append("### Dumps")
+                markdown.extend(dumps_section)
+        else:
+            if isinstance(identities, dict):
+                identities = identities.get("login")
+            markdown = [
+                f'## There is no data for {", ".join(identities)} in selected period'
+            ]
+
+        return "\n".join(markdown)
+
+    def __build_lookup_context(self, lookup_data, identities):
+        result = CommandResults(
+            readable_output=self.__build_lookup_markdown(
+                lookup_data, identities
+            ),
+            outputs_prefix="RecordedFuture.Credentials.Identities",
+            outputs=lookup_data,
+        )
+        return result
+
+
+def main() -> None:
+    """Main method used to run actions."""
+    try:
+        demisto_params = demisto.params()
+        demisto_args = demisto.args()
+        base_url = demisto_params.get("server_url", "").rstrip("/")
+        verify_ssl = not demisto_params.get("unsecure", False)
+        proxy = demisto_params.get("proxy", False)
+        # Clean the input from the user and split string
+        domains = (
+            demisto_params.get("domains", "")
+            .rstrip(";")
+            .replace(" ", "")
+            .split(";")
+        )
+        # If user has not set password properties we will get empty string but client require empty list
+        password_properties = demisto_params.get("password_properties") or []
+        try:
+            limit_identities = int(
+                demisto_params.get("limit_identities", LIMIT_IDENTITIES)
+            )
+            # We can't do that on UI, so set the limit here.
+            if limit_identities > LIMIT_IDENTITIES:
+                limit_identities = LIMIT_IDENTITIES
+        except ValueError:
+            limit_identities = LIMIT_IDENTITIES
+
+        headers = {
+            "X-RFToken": demisto_params["token"],
+            "X-RF-User-Agent": f"xsoar-identity/{__version__} rfclient (Cortex_XSOAR_"
+            f'{demisto.demistoVersion()["version"]})',
+        }
+        client = Client(
+            base_url=base_url,
+            verify=verify_ssl,
+            headers=headers,
+            proxy=proxy,
+        )
+        command = demisto.command()
+        actions = Actions(client)
+        if command == "test-module":
+            try:
+                client.whoami()
+                return_results("ok")
+            except Exception as err:
+                message = str(err)
+                try:
+                    error = json.loads(str(err).split("\n")[1])
+                    if "fail" in error.get("result", {}).get("status", ""):
+                        message = error.get("result", {})["message"]
```

Comment:
```suggestion
                        message = error.get("result", {}).get("message",<some default message>)
```

to avoid KeyError on accessing `{}["message"]`


on a second thought, catching it on the next line like you did, is also an option, both are good.